jupyter_events/cli.py

```diff
@@ -1,5 +1,6 @@
 import json
 import pathlib
+import traceback
```

Comment:
this should be removed, since it's not used in this module.

Reply:
Again, no warning from linter -- just a slightly greyed out import on my editor 🙈